examples/win32-front-proxy/Dockerfile-service

```diff
@@ -1,4 +1,4 @@
-FROM envoyproxy/envoy-windows-dev:latest
+FROM python:3.8-slim-bullseye
```

Comment:
not sure about this one - i think this needs to be win image - there are python win images so potentially we could use one of them - but no way to debug etc so may be easier to just leave this as it is

Reply:
OK, just leave it.